broadinstitute/ddp-study-server
# DSM-7581: Adding data dictionary capability to RC

This adds data dictionary support and also cleans up yet more potential null pointer edge cases.  The data dictionary is a complete listing of all study variables, useful for researchers doing work on the data set.  Pressing "download" will now download a .zip file of both the exported data, and a data dictionary excel document.  See the file below for an example data dictionary generated off of development AT

[DataDictionary.xlsx](https://github.com/broadinstitute/ddp-study-server/files/9271277/DataDictionary.xlsx)

TO TEST:
1. load a study participant list
2. select "download"
3. download with the default options
4. observe that a zip file is downloaded, containing both a PArticipant data export, and a "DataDictionary.xslx" file
